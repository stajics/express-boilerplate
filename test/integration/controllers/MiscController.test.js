require('../../index');

describe(':misc', () => {
  jestTest('[POST /notFound] Should get 404', async (done) => {
    const res = await client.post('/notFound', {
      email: 'test1@test.com',
      firstName: 'name',
      lastName: 'name',
    });
    expect(res.body).toMatchSnapshot();
    done();
  });
});
