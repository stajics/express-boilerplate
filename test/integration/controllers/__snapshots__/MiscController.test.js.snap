// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`:misc [POST /notFound] Should get 404 1`] = `
Object {
  "error": Object {
    "status": 404,
  },
}
`;
